pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/LoadManager.java

```diff
@@ -79,6 +83,11 @@
     void doNamespaceBundleSplit() throws Exception;
 
     /**
+     * Determine the broker root.
+     */
+    String getBrokerRoot();
```

Comment:
should we name it as : `getBrokerRootName()` as we are returning name rather root node?